### PR TITLE
rmw 0.4.03 (new formula)

### PR DESCRIPTION
**Pull request by martijndeb**
Adds new formula for the application rmw (ReMove to Waste), which is a cross platform utility allowing you to safely remove files via a trash folder.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/rmw.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class Rmw < Formula
+  desc "OS portable cli trash can utility written in C"
+  homepage "https://github.com/andy5995/rmw/wiki"
+  url "https://github.com/andy5995/rmw/archive/v0.4.01.tar.gz"
```
Comment (andy5995) [inline Formula/rmw.rb]: I'm the maintainer of rmw.

Thanks for submitting this @sexybiggetje 

This link my cause a problem. It redirects to a file with a different name (rmw-0.4.01.tar.gz).

If there's any problem, you can use this link instead:
https://github.com/andy5995/rmw/releases/download/v0.4.01/rmw-0.4.01.tar.gz

Comment (sexybiggetje) [reply Formula/rmw.rb]: The link is as used in the Formula cookbook, and local installation from that link works fine.
Wonder why Jenkins fails below tho.

**Comment by DomT4** [comment]
> Wonder why Jenkins fails below tho.

```
09:53:14 rmw:
09:53:14   * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
```

It's kinda buried within the output but that's the failure.

**Comment by andy5995** [comment]
>It's kinda buried within the output but that's the failure.

Thank you for clearing that up @DomT4 .

**Comment by ilovezfs** [comment]
Thanks for the PR @sexybiggetje. You may want to host it in [your own tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md#how-to-create-and-maintain-a-tap) for now.

The formula will need a test that exercises and verifies the functionality of the software as well.

**Pull request closed without merging**